iree/compiler/Dialect/VM/Transforms/Passes.h

```diff
@@ -77,6 +77,11 @@ createOrdinalAllocationPass();
 std::unique_ptr<OperationPass<IREE::VM::ModuleOp>>
 createConvertStandardToVMPass();
 
+std::unique_ptr<OperationPass<IREE::VM::FuncOp>> createValueLivenessTestPass();
```

Comment:
I think we should probably give these their own header, since they're not really transforms and they're test passes. Can we give them a `TestPasses.h` file in Analysis/

Reply:
Sure :)